src/transport/raw/tests/BUILD.gn

```diff
@@ -1,23 +1,22 @@
-# Copyright (c) 2020 Project CHIP Authors
```

Comment:
Why this reformatting?

Reply:
Accident (I ran the GN file through clang formatter).  The comments have been restored to their previous formatting.